src/__internal__/focus-trap/focus-trap-utils.js

```diff
@@ -0,0 +1,42 @@
+const defaultFocusableSelectors =
+  'button:not([disabled]), [href], input:not([type="hidden"]):not([disabled]), select:not([disabled]), textarea:not([disabled]), [tabindex]';
+
+function setElementFocus(element) {
+  if (typeof element === "function") {
+    element();
+  } else {
+    const el = element.current || element;
+    el.focus();
+  }
+}
+
+const isRadio = (element) => {
+  return (
+    element.hasAttribute("type") && element.getAttribute("type") === "radio"
```

Comment:
Shouldn't we just have ` element.getAttribute("type") === "radio"`
Or we should change the function name if we want to have range out of radio

Reply:
So this is saying "if the element has a `type` attribute AND the type is `radio`" then return true - we need the first check otherwise we don't know if there's a `type` attribute to check. 

So the function name makes sense because we're only checking if the element is a radio button